codec-http2/src/test/java/io/netty/handler/codec/http2/Http2StreamChannelBootstrapTest.java

```diff
@@ -114,9 +110,12 @@ public void run() {
             assertThat(promise.isDone(), is(false));
             closeLatch.countDown();
 
-            exceptionRule.expect(ExecutionException.class);
-            exceptionRule.expectCause(IsInstanceOf.<Throwable>instanceOf(ClosedChannelException.class));
-            promise.get(3, SECONDS);
+            try {
+                promise.get(3, SECONDS);
+                fail();
+            } catch (ExecutionException e) {
+                assertThat(e.getCause(), IsInstanceOf.<Throwable>instanceOf(ClosedChannelException.class));
+            }
```

Comment:
If you use `assertThrows`, it'll return the exception thrown for further checks.

Reply:
TIL... Let me refactor

Reply:
Out of context question, What does `TIL` mean?

Reply:
@hyperxpro Today I Learned